packages/react/src/components/table/Table.module.scss

```diff
@@ -0,0 +1,117 @@
+.table {
+  composes: hds-table from 'hds-core/lib/components/table/table.css';
+
+  &.dark th {
+    color: var(--color-white);
+  }
+
+  &.light th {
+    color: var(--color-black-90);
+  }
+}
+
+.caption {
+  composes: hds-table__caption from 'hds-core/lib/components/table/table.css';
+}
+
+.container {
+  composes: hds-table-container from 'hds-core/lib/components/table/table.css';
+  height: inherit;
+}
+
+.headerRow {
+  composes: hds-table__header-row from 'hds-core/lib/components/table/table.css';
+}
+
+.content {
+  composes: hds-table__content from 'hds-core/lib/components/table/table.css';
+}
+
+.dark {
+  --background-color: var(--color-bus);
```

Comment:
Why not just compose the hds-table--dark from core? The reason was that I could then not get the custom theme for background color for react version of the table to work. Maybe we should not use the `useTheme` at all here? User could just pass a className where he overrides the variable? Problem with that is, that it is harder to demonstrate in a story, as passing a className hides the content of the className, which would be the beef here. 🤔